### DISCUSSION TITLE
I created a rule based bot - Genie

### DISCUSSION THREAD

**Discussion by sureshpandiyan1** [explosion/spaCy#11832] [Show & Tell]
Hello Spacy Team,

i am happy to say about my repo build with spacy - Genie (desktop application), actually spacy is not only train the model, 
even we can make something cool with spacy...i really love with spacy  :)

# Intro
Genie is a rule-based bot  desktop application. It's a helpful desktop application for everyone
who wants to do simple things quickly like download a file, minify, etc., 

It's just on pre-release v1.0.0

[genie_edit_0.webm](https://user-images.githubusercontent.com/112636345/202832108-78c4dcfe-cf61-48d2-81dc-da0fbf410b13.webm)

There is lot of things, i like to say about genie,
you can take a look at above video and my github - readme.md too
even you can click download button on readme to check my genie too :)

https://github.com/sureshpandiyan1/Genie

@ines 
special thanks to ines for the course about advanced nlp with spacy :)
it's takes 6 days to learn your course and put full efforts on it, but finally i build something cool with spacy :)
i am really proud to share with you and spacy too :)

@polm  @philipvollet 

I am really looking forward for a feedback what I built with spacy :)

**Comment by polm** [comment]
Congrats on your project!

I had some trouble understanding what the valid commands were. Maybe you could add some documentation about them? I see there's a list in the PDF but it doesn't say what they actually do. Also note you can link to PDFs directly on Github instead of using Mediafire.

**Comment by sureshpandiyan1** [comment]
@polm 

okey...just now before i updated my whole documentation :)
even i changed mediafire instead of github link for docs too..

i did everything what you are said too me. 

Thank you for guided me :)

**Reply by polm** [reply]
Thanks! I think the new docs make it easier to understand.

**Reply by sureshpandiyan1** [reply]
@polm 

even i already tag my project on twitter too..
still time also, i don't get any response from "@spacy_io"

i written a detailed documentation about genie :)

i like to be example for everyone who's learn a course of spacy and build something with that :)
